lib/src/model/nutrient_modifier.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:openfoodfacts/src/model/off_tagged.dart';
+
+enum NutrientModifier implements OffTagged {
+  approximately,
+  superiorTo,
+  inferiorTo;
+
+  @override
+  String get offTag => symbol;
+}
```

Comment:
```suggestion
enum NutrientModifier implements OffTagged {
  approximately(offTag: '~'),
  greaterThan(offTag: '>'),
  lessThan(offTag: '<');

  const NutrientModifier({
    required this.offTag,
  });

  @override
  final String offTag;

  /// Returns the first [NutrientModifier] that matches the [offTag].
  static NutrientModifier? fromOffTag(final String? offTag) =>
      OffTagged.fromOffTag(offTag, NutrientModifier.values) as NutrientModifier?;
}
```

I'm not saying what you coded is wrong, but what about replicating the way we code other `OffTagged`s?

Btw in English they rather say `greaterThan` and `lessThan` ;)